scriptworker/test/test_context.py

```diff
@@ -124,3 +126,23 @@ def test_get_credentials(context):
     expected = {'asdf': 'foobar'}
     context._credentials = expected
     assert context.credentials == expected
+
+
+def test_new_event_loop(mocker):
+    """The default context.event_loop is from `asyncio.get_event_loop`"""
+    fake_loop = mock.MagicMock()
+    mocker.patch.object(asyncio, 'get_event_loop', return_value=fake_loop)
+    context = swcontext.Context()
+    assert context.event_loop == fake_loop
```

Comment:
Nit: I think we should use the `is` operator in this case. Thus, we're sure this is the same object.

Reply:
Makes sense! Done.